Packages/com.unity.inputsystem/InputSystem/Editor/InputActionCodeGenerator.cs

```diff
@@ -472,6 +472,13 @@ public static bool GenerateWrapperCode(string filePath, IEnumerable<InputActionM
                     return false;
             }
 
+            if (!filePath.ToLower().StartsWith("assets/"))
+                filePath = Path.Combine("Assets", filePath);
+
+            var dir = Path.GetDirectoryName(filePath);
+            if (!Directory.Exists(dir))
+                Directory.CreateDirectory(dir);
```

Comment:
Do these steps in importer instead of here. No need to hardcode the code generator in terms of where it writes the file to.